R-packages/covidcast/NEWS.md

```diff
@@ -1,3 +1,14 @@
+# covidcast 0.4.0
+
+Released TODO DATE.
```

Comment:
Does this need a real date now?

Reply:
No, it shouldn't; it needs a real date when we cut the 0.4.0 release by merging r-pkg-devel to main.